### DISCUSSION TITLE
Coolify not running after restart

### DISCUSSION THREAD

**Discussion by vitezprchal** [coollabsio/coolify#3829] [Q&A]
Hello,
After a recent forced reboot, coolify instance is no longer running. I have attempted basic troubleshooting but haven’t been able to resolve the issue. Could you kindly provide guidance on how to restart the instance or direct me to any relevant documentation that could help resolve this problem?

**Comment by Procuria** [comment]
@vitezprchal Coolify itself is running as a Docker Compose project, so any tool which provides you access to Docker logs should be helpful to you. 

You could [set up Portainer](https://docs.portainer.io/start/install-ce/server/docker/linux) or [Dozzle](https://dozzle.dev/guide/getting-started) and check the container logs there.  
You can use the 'native' [docker logs](https://docs.docker.com/reference/cli/docker/container/logs/) command also. 

This should give you at least a starting point to investigate further. 

You can use the `/data/coolify/source/upgrade.sh` for "restarting" the project.
Be aware that you can add the desired Coolify version as argument 
````bash
./update.sh  4.0.0-beta.xxx 
````
